### PR TITLE
Remove stale hyperlinks

### PR DESCRIPTION
**Pull request by louquillio**
Remove/update some broken link targets (#131)
    
These stale outbound links seem safe to remove, update or annotate.
    
The remaining stale targets *may* have valuable replacements that I'm unable to identify. Someone more knowledgeable should take a look.

### CODE REVIEW

File: _posts/2012-03-05-source-code-to-apprtc.appspot.com-example-app-available.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ date: 2012-03-05 14:21:00
 
 
 The source code to <https://apprtc.appspot.com/> is now available at
-<https://chromium.googlesource.com/external/webrtc/+/master/talk/examples/android/src/org/appspot/apprtc/>
+https://chromium.googlesource.com/external/webrtc/+/master/talk/examples/android/src/org/appspot/apprtc/ (link outdated).
```
Comment (x1ddos) [inline _posts/2012-03-05-source-code-to-apprtc.appspot.com-example-app-available.md]: I believe this is now on github, no? https://github.com/webrtc/apprtc

**Comment by x1ddos** [comment]
Only apprc links needs updating, I think. The rest looks good.

File: _posts/2012-03-05-source-code-to-apprtc.appspot.com-example-app-available.md (hunk 1 above)
Comment (x1ddos) [reply _posts/2012-03-05-source-code-to-apprtc.appspot.com-example-app-available.md]: Actually, that's probably not right. This is the one: https://chromium.googlesource.com/external/webrtc/+/master/webrtc/examples/androidapp/src/org/appspot/apprtc

Comment (MirkoBonadei) [reply _posts/2012-03-05-source-code-to-apprtc.appspot.com-example-app-available.md]: Yes, the second link posted by @x1ddos is the right one.

**Comment by MirkoBonadei** [comment]
Thanks for doing this @louquillio!

**Pull request merged**